gh-actions/go/gotestfmt/action.yaml

```diff
@@ -13,7 +13,7 @@ runs:
       run: |
         # Install gotestfmt
         set -eu
-        TOOL=github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt
+        TOOL=github.com/adombeck/gotestfmt/v2/cmd/gotestfmt
```

Comment:
Maybe use it from a specific branch? So that your fork can change without breaking the actions?

Or what about moving it to a canonical-owned fork instead? As I don't feel `gotestfmt` will handle this very soon...

Reply:
> Or what about moving it to a canonical-owned fork instead? As I don't feel gotestfmt will handle this very soon...

Yeah I also thought about that. Works for me, but I don't think I have the permission to create a fork under the canonical namespace

Reply:
that's done now